Code/EnginePlugins/AngelScriptPlugin/TypeRegistration/AsRegister_Reflected.cpp

```diff
@@ -5,17 +5,18 @@
 #include <AngelScriptPlugin/Utils/AngelScriptUtils.h>
 #include <Foundation/Reflection/ReflectionUtils.h>
 
-bool ezAngelScriptEngineSingleton::AppendType(ezStringBuilder& decl, const ezRTTI* pRtti, const ezScriptableFunctionAttribute* pFuncAttr, ezUInt32 uiArg, bool& inout_VarArgs)
+bool ezAngelScriptEngineSingleton::AppendType(ezStringBuilder& decl, const ezRTTI* pRtti, const ezScriptableFunctionAttribute* pFuncAttr, ezUInt32 uiArg)
 {
   const bool bIsReturnValue = uiArg == ezInvalidIndex;
+  const auto argType = pFuncAttr ? pFuncAttr->GetArgumentType(uiArg) : ezScriptableFunctionAttribute::ArgType::In;
 
-  if (pRtti == nullptr)
+  if (pRtti == nullptr || pRtti == ezGetStaticRTTI<ezVariantArray>())
```

Comment:
If bIsReturnValue == false, this shouldn't append void, but probably do nothing or assert.
I guess we don't have that case, because ezVariantArray is (so far) only used as a last vararg parameter.